ores/wsgi/util.py

```diff
@@ -182,8 +182,7 @@ def build_event_set(event):
     uses internally.
     """
     event_set = set()
-    if re.match(r"([^\.]+.)?mediawiki\.revision-create$",
-                event['meta']['topic']):
+    if event['meta']['schema'] == "mediawiki.revision-create":
```

Comment:
I'm wondering if that check is needed at all. Changeprop is configured to only update ORES on `revision-create`, so nothing else could be sent here.

Reply:
Eventually, we'll want to be able to precache based on other types of events.  So I'd like to keep it here.  Either way, it's good practice to check our assumption explicitly.